packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -116,32 +111,75 @@ export class Meter implements api.Meter {
   }
 
   /**
-   * Creates a new observer metric.
+   * Creates a new value observer metric.
    * @param name the name of the metric.
    * @param [options] the metric options.
+   * @param [callback] the value observer callback
    */
-  createObserver(name: string, options?: api.MetricOptions): api.Observer {
+  createValueObserver(
+    name: string,
+    options: api.MetricOptions = {},
+    callback?: (observerResult: api.ObserverResult) => void
+  ): api.ValueObserver {
+    if (!this._isValidName(name)) {
+      this._logger.warn(
+        `Invalid metric name ${name}. Defaulting to noop metric implementation.`
+      );
+      return api.NOOP_VALUE_OBSERVER_METRIC;
+    }
+    const opt: api.MetricOptions = {
+      monotonic: false,
+      absolute: false, // not applicable to observer, set to false
+      logger: this._logger,
+      ...DEFAULT_METRIC_OPTIONS,
```

Comment:
Any values in DEFAULT_METRIC_OPTIONS will override the three hard coded options you put (monotonic, absolute, and logger). Is that intentional?

Reply:
fixed